content/topics/intro-to-relational-databases/_index.md

```diff
@@ -1,15 +1,69 @@
 ---
-title: SQL
-ready: True
+title: Introduction to relational database
+ready: true
 ---
 
-## Tutorial
+# INTRODUCTION TO RELATIONAL DATABASE 
 
-Complete the DataCamp Intro and Intermediate SQL courses:
+**LESSONS**
 
+- Lesson 1: 
+  > Data and Tables:
+    * learn the principles behind relational data organization:tables,queries,aggregations,
+    keys and joins.
```

Comment:
where can one find these lessons? Are the recruits supposed to just google them?

Provide useful links